sdk.sln

```diff
@@ -946,6 +947,10 @@ Global
 		{FAADC193-BA41-449D-97CE-0EF82836046A}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{FAADC193-BA41-449D-97CE-0EF82836046A}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{FAADC193-BA41-449D-97CE-0EF82836046A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{8D6A9984-118D-4415-A8FA-AB1F26CF5C44}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
what are these?

Reply:
They got reordered 
![image](https://github.com/user-attachments/assets/9e999fda-f0c1-4b33-a1ef-9b506fd18ce5)
Should be the same

Reply:
Same as the next diff

Reply:
Removed as to avoid potential git conflicts :)